cfgov/unprocessed/js/molecules/GlobalSearch.js

```diff
@@ -122,11 +142,39 @@ function GlobalSearch( element ) {
    * which opens/closes the search input.
    */
   function _triggerClicked() {
-    if ( _isExpanded ) {
-      collapse();
-    } else {
-      expand();
-    }
+    this.dispatchEvent( 'triggerClick', { target: event.target } );
+  }
+
+  /**
+   * Event handler for when FlyoutMenu expand transition begins.
+   * Use this to perform post-expandBegin actions.
+   */
+  function _handleExpandBegin() {
+    _contentDom.classList.remove( 'u-hidden' );
+
+    // TODO: Refactor to remove totally infuriating
+    //       bug where select() won't fire without timeout.
```

Comment:
The CSS would need to be changed from a transition to an animation I think. I can explore in another PR.